mysql/assets/configuration/spec.yaml

```diff
@@ -369,9 +369,9 @@ files:
           - name: obfuscator_options
             description: Configure how the SQL obfuscator behaves.
             options:
-              - name: quantize_sql_tables
+              - name: replace_digits
```

Comment:
Is the updated option supposed to be `replace_digits` or `quantize_sql`?

Reply:
My mistake, didn't update the PR body. It should be `replace_digits` after some discussion in the agent PR https://github.com/DataDog/datadog-agent/pull/8860